qiskit/aqua/algorithms/algorithm_result.py

```diff
@@ -45,7 +45,7 @@ def pop(self, key: object, default: Optional[object] = None) -> object:
     def popitem(self) -> Tuple[object, object]:
         raise TypeError("'popitem' invalid for this object.")
 
-    def update(self, *args, **kwargs) -> None:
+    def update(self, *args, **kwargs) -> None:  # pylint: disable=signature-differs
```

Comment:
I guess it is not related to the PR. Why is this change?

Reply:
pylint warning

Reply:
Is it your local issue or from the build pipeline? On my laptop I don't have any issues here.

Reply:
@adekusar-drl This I think arises when linting under 3.8 which we do not do in the build. I asked @manoelmarques as the original author to look at this and this was his finding. Since this overrides a method defined in UserDict it maybe that signature was updated (the one I looked at certainly was bit different)

Reply:
@woodsp-ibm Thanks a lot for the clarification! I'm still on 3.7 locally.

Reply:
Also, the msg "signature-differs" is from the latest pylint. The pylint version used by Aqua, Ignis etc is 2.4.4 and with that version the msg is "arguments-differ". We should decide if forcing 2.4.4 is still necessary, if that is the case it may need to be changed to: # pylint: disable=signature-differs,arguments-differ